City-of-Helsinki/berth-reservations
# VEN-813 | Attach lease to WS section instead of WS area

## Description :sparkles:
Attach lease to WS section instead of WS area

## Issues :bug:
### Closes :no_good_woman:
[VEN-813](https://helsinkisolutionoffice.atlassian.net/browse/VEN-813)

## Testing :alembic:
### Automated tests :gear:️
- There were already quite nice bunch of integration tests related to creating a lease with area, so converted those tests to use section instead
### Manual testing :construction_worker_man:
- Go to http://localhost:8000/admin/leases/winterstoragelease/
- Add new lease with section assigned
- Fire a query via `graphql_v2`
```
query WinterStorageSections {
  winterStorageSections {
    edges {
      node {
        id
        properties {
          leases {
            edges {
              node {
                id
                isActive
                customer {
                  id
                }
              }
            }
          }
        }
      }
    }
  }
}
```
You should see the lease in the response.
